skype/skype-ps/skype/Get-CsOnlineAudioFile.md

```diff
@@ -0,0 +1,119 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml 
+online version: https://docs.microsoft.com/powershell/module/skype/get-csonlineaudiofile
+applicable: Microsoft Teams
+author: jenstrier
+ms.author: jenstr
+ms.reviewer: 
+manager:
+schema: 2.0.0
+---
+
+# Get-CsOnlineAudioFile
+
+## SYNOPSIS
+Returns information about a specific or all uploaded audio files of a given application type.
+
+> [!NOTE]
+> **Preview** The use of this cmdlet is in Public Preview.
+  
+## SYNTAX
+
+```powershell
+Get-CsOnlineAudioFile [[-Identity] <string>] [-ApplicationId <OrgAutoAttendant | HuntGroup | TenantGlobal>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet returns information on a specific or all uploaded audio files of a given application type. If you are not specifying any parameters you will get information of all uploaded audio files with ApplicationId = TenantGlobal.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Get-CsOnlineAudioFile
+
+Id                : 85364afb59a143fc9466979e0f34f749
+FileName          : CustomMoH.mp3
+ApplicationId     : TenantGlobal
+MarkedForDeletion : False
+```

```

Comment:
For examples with output, please use the following format:
https://docs.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide#executable-examples